### PR TITLE
Can't create evm chain wallet in test network

### PR DESCRIPTION
**Pull request by shuaiLiWang**
## What does this implement/fix? Explain your changes.
测试网不能创建eth类钱包

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
…

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
python

## Any other comments?
…

### CODE REVIEW

**Review by taimanhui — DISMISSED** [review]

**Comment by taimanhui** [comment]
need rebase

**Comment by shuaiLiWang** [comment]
> need rebase

ack

**Review by liyanhrxy — APPROVED** [review]

**Review by taimanhui — APPROVED** [review]

**Review by liyanhrxy — DISMISSED** [review]

**Review by liyanhrxy — APPROVED** [review]

File: electrum_gui/android/console.py [hunk 1]
```diff
@@ -3038,14 +3038,16 @@ def create_hd_wallet(
                     hd=True,
                 )
             elif chain_affinity == "eth":
+                chain_code_main = coin_manager.chain_code_to_legacy_coin(chain_info.chain_code)
+                coin = coin_manager.get_coin_info(chain_info.fee_code)
                 wallet_info = self.create(
-                    coin.upper(),
+                    f"{coin.symbol}",
```
Comment (taimanhui) [inline electrum_gui/android/console.py:3044]: 这个不太对，需要传进去的是名字，不一定是币种符号，比如接下来会出问题的OEC和OKT。

**Review by taimanhui — COMMENTED** [review]
这个问题好像已经没有了，因为 #739 改这里的时候把问题给处理掉了

File: electrum_gui/android/console.py (hunk 1 above)
Comment (shuaiLiWang) [reply electrum_gui/android/console.py:3044]: 这块感觉太乱了 我去问下设计 这块到底显示啥

**Comment by taimanhui** [comment]
> 测试网为什么会不能创建eth类钱包？我本地可以呀。应该是 set_testnet 和 IS_DEV 初始化时机错误的问题

wallet manager的PR(#739)修了，你关注下那里对create_hd_wallet的修改，之前是遍历了eth_affinity的链再检查，然后遍历出来的都带前缀`t`所以就出问题了。你刚好改到这里把它改成直接取affinity再检查对上eth了。😂

**Comment by huazhouwang** [comment]
懂了

**Pull request closed without merging**